surf/apps/filters/models.py

```diff
@@ -58,6 +61,44 @@ class FilterCategoryItem(UUIDModel):
     def __str__(self):
         return self.title
 
+class MpttFilterItem(MPTTModel, UUIDModel):
+    name = django_models.CharField(max_length=255)
+    parent = TreeForeignKey('self', on_delete=django_models.CASCADE, null=True, blank=True, related_name='children')
+
+    created_at = django_models.DateTimeField(auto_now_add=True)
+    updated_at = django_models.DateTimeField(auto_now=True)
+    deleted_from_edurep_at = django_models.DateTimeField(default=None, null=True, blank=True)
+
+    title_translations = django_models.OneToOneField(to=Locale, on_delete=django_models.CASCADE, null=True, blank=False)
+    external_id = django_models.CharField(max_length=255, verbose_name="Field id in EduRep", blank=True)
+    enabled_by_default = django_models.BooleanField(default=False)
+    is_hidden = django_models.BooleanField(default=False)
+
+    item_count = 0
+
+    def __str__(self):
+        return self.name
+
+    def to_dict(self):
```

Comment:
Deze mag weg neem ik aan? Dit wordt geregeld door de serializer is mijn verwachting